portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -83,13 +87,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty(IMAGE_URL, uploadedUrl.isPresent() ? uploadedUrl.get() : "");
     commentEntity.setProperty(COMMENT_TEXT, commentText);
     commentEntity.setProperty(TIME_STAMP, timeStamp);
+    commentEntity.setProperty(SENTIMENT_SCORE, getSentiment(commentText));
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(commentEntity);
 
     response.sendRedirect(REDIRECT_URL);
   }
 
+  /**
+   * Returns sentiment analysis score of {@code commentText}.
```

Comment:
This method should return what the range is and means.

Reply:
fixed